src/main/controllers/helpers.ts

```diff
@@ -265,6 +265,18 @@ export const conditionalRedirect = (
   return matchingValues?.some(v => v === condition);
 };
 
+export const handleUpdateDraftCase = (req: AppRequest, logger: LoggerInstance): void => {
+  if (!req.session.errors.length) {
```

Comment:
if we invert this if statement we can have an early return, that'll remove one level of indentation.
```
if (req.session.errors.length) {
 return;
}

// getCaseApi code
 ```
 
 Also is it possible to use `async/await` here instead of using `.then` and `.catch`?